app/src/main/java/com/github/se/eduverse/ui/gallery/GalleryScreen.kt

```diff
@@ -123,13 +135,18 @@ fun ImageDialog(photo: Photo, onDismiss: () -> Unit, onDownload: () -> Unit) {
             }
       },
       dismissButton = {
-        IconButton(
-            onClick = { onDownload() },
-            modifier =
-                Modifier.testTag("DownloadButton") // Ajout du testTag pour le bouton Download
-            ) {
-              Icon(imageVector = Icons.Default.Download, contentDescription = "Download Image")
-            }
+        Row {
+          IconButton(
+              onClick = { onDownload() },
+              modifier =
+                  Modifier.testTag("DownloadButton") // Ajout du testTag pour le bouton Download
+              ) {
+                Icon(imageVector = Icons.Default.Download, contentDescription = "Download Image")
+              }
+          TextButton(onClick = { onAdd() }, modifier = Modifier.testTag("addToFolder")) {
```

Comment:
I think that the composables representing the different actions should be defined in the body of the body of the AlertDialog, this parameter is meant to customize the dismiss button of the dialog and I think it's confusing to use it this way. Maybe we should consider refactoring this code in another task

Reply:
AlertDialog do not have a "body", simply a "text" and 2 buttons. I decided to do things this way cause it seemed better than redoing the whole dialog to get to the same result, but if you thing this is an improvement I will do it.

Reply:
Yes sorry I thought the AlertDialog had a body